index.template.html

```diff
@@ -5141,6 +5141,13 @@ <h3>Extension and Import</h3>
     </span>
     </p>
 
+    <p class="note">
+        Portions of the <code>tm:ref</code> value might contain non-ASCII characters that require URL ("percent") 
+        encoding before use. Before applying escapes to a <code>tm:ref</code> value, implementations should check that the value is not already 
```

Comment:
I think we need an RFC assertion here while making sure that the assertion is understandable on its own (implies changes to the sentence)